posts/2021-09-29-jcon-conference.adoc

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title: "MicroProfile, Jkarta EE and Open Liberty at JCON-Online 2021"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/GraceJansen
+author_github: https://github.com/GraceJansen
+seo-title: MicroProfile, Jkarta EE and Open Liberty at JCON-Online 2021 - OpenLiberty.io
+seo-description: A helpful summary of the JCON-Online 2021 workshops and talks on MicroProfile, Jakarta EE and Open Liberty.
+blog_description: "A helpful summary of the JCON-Online 2021 workshops and talks on MicroProfile, Jakarta EE and Open Liberty."
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= MicroProfile, Jakarta EE and Open Liberty at JCON-Online 2021
+Grace Jansen <https://github.com/GraceJansen>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Despite the global pandemic and general chaos of this year, many amazing developer events and conferences have battled through are still taking place, bring the developer community together once more - many virtually! JCON-Online is one such event. It's aiming to really utilise this digital format to bring together the international Java community. The great part is, it's free for all JUG members!
+
+Several of the Open Liberty team will be joining https://jcon.one/[JCON-Online 2021] on 5-8 October. If you're also going to be there, deciding which of the great talks to see at JCON looks like it could be a challenge. So we've put together a helpful summary of the sessions on https://microprofile.io/[MicroProfile], https://jakarta.ee/[Jakarta EE] and https://openliberty.io/about/[Open Liberty].
```

Comment:
Several of the Open Liberty team
--->
Several members of the Open Liberty team
OR
Several of us from the Open Liberty team